core/src/wsv.rs

```diff
@@ -335,15 +335,34 @@ impl WorldStateView {
     /// - If timestamp conversion to `u64` fails
     #[iroha_logger::log(skip_all, fields(block_height))]
     pub fn apply(&mut self, block: &VersionedCommittedBlock) -> Result<()> {
```

Comment:
Is it used anywhere after your changes?

Reply:
We use it in some test where we don't validate block.